src/Concrete/Entity/AttributeForm.php

```diff
@@ -289,7 +290,7 @@ public function getAsHtml()
             $label = $ak->getAttributeKeyDisplayName();
             $value = $this->getAttribute($ak, 'display');
 
-            $submittedDataHtml .= '<tr><th>' . h($label) . '</th><td>' . h($value) . '</td></tr>';
+            $submittedDataHtml .= '<tr><th>' . HtmLawed::htmLawed($label) . '</th><td>' . HtmLawed::htmLawed($value) . '</td></tr>';
```

Comment:
if we would like a scripting (like javascript) validation, it will be useful. i will add it